parquet/src/test/java/org/apache/iceberg/avro/TestReadProjection.java

```diff
@@ -30,17 +31,15 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Sets;
 import org.apache.iceberg.types.Comparators;
 import org.apache.iceberg.types.Types;
-import org.assertj.core.api.Assertions;
-import org.junit.Assert;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
+import org.junit.jupiter.api.Assertions;
```

Comment:
please use AssertJ-style assertions instead of the Junit ones

Reply:
Changed to use AssertJ style assertions